jobs/JGLOBAL_PREP_AERO_OBS

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export DATA=${DATA:-${DATAROOT}/${RUN}aeroanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "prepaeroobs" -c "base aeroanl prepaeroobs"
+
+##############################################
+# Set variables used in the script
+##############################################
+# shellcheck disable=SC2153
+GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
+gPDY=${GDATE:0:8}
+gcyc=${GDATE:8:2}
+GDUMP="gdas"
+
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS COM_CHEM_ANALYSIS
```

Comment:
`COM_OBS` should be considered read-only. In development we copy obs from the dump archive to a location within our directories, but in operations we use them directly from obsproc. We can make new prep COM variables in [config.com](https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/config/gfs/config.com) for any components that need them.

@CoryMartin-NOAA @guillaumevernieres I think there are other jobs recently that have the same issue that we just let through that will need to be updated.

Reply:
I think we need to have a bigger discussion on this. There are obs for the non-atmospheric components that are not being processed through the traditional obsproc channels (in particular the aerosol and SOCA datasets). I'll send an email to follow up and schedule a meeting.

Reply:
My solution would be to create a `COM_<component>_PREP_TMPL` (or similar; waves already has a prep) to store pre-processed stuff.